src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php

```diff
@@ -20,9 +20,12 @@ class ScreenshotCompareExtension implements ExtensionInterface
     /**
      * @var AdapterFactory[]
      */
-    private $adapterFactories = array();
+    private $adapterFactories = [];
```

Comment:
The new array syntax is only supported in PHP 5.4+. Please adjust the composer accordingly.